repos/spack_repo/builtin/packages/py_mdanalysis/package.py

```diff
@@ -38,14 +42,17 @@ class PyMdanalysis(PythonPackage):
     depends_on("c", type="build")  # generated
     depends_on("cxx", type="build")  # generated
 
+    depends_on("python@3.10:", type=("build", "run"), when="@2.8.0:")
     depends_on("python@3.9:", type=("build", "run"), when="@2.5.0:")
     depends_on("python@3.8:", type=("build", "run"))
 
     depends_on("py-setuptools", type="build")
+    depends_on("py-setuptools@40.9:", type="build", when="@2.8.0:")
 
     depends_on("py-cython@0.28:", type="build")
 
     # MDAnalysis required dependencies (install_requires)
+    depends_on("py-numpy@2:", when="@2.8.0:", type=("build", "run"))
```

Comment:
@tldahlgren I've merged the `numpy@2:` and `numpy@1.23:` constraint to just `numpy@2:`.